Extension/src/LanguageServer/extension.ts

```diff
@@ -1062,6 +1110,76 @@ function handleMacCrashFileRead(err: NodeJS.ErrnoException | undefined | null, d
     logMacCrashTelemetry(data);
 }
 
+async function handleCrashFileRead(crashDirectory: string, crashFile: string, err: NodeJS.ErrnoException | undefined | null, data: string): Promise<void> {
+    if (err) {
+        if (err.code === "ENOENT") {
+            return; // ignore known issue
+        }
+        return logCppCrashTelemetry("readFile: " + err.code);
+    }
+
+    console.log("processing crash log");
```

Comment:
I'm not sure about this console.log yet...should I change it?

Reply:
`console.log` doesn't show up in the Output window, so you are welcome to leave it like this if you want.

Reply:
Oh -- it does show up in the "Server" output pane, but that seems fine to me. I remove "processing crash log" though since that was not useful logging.